webview/src/experiments/components/table/Indicators.tsx

```diff
@@ -123,6 +134,15 @@ export const Indicators = () => {
       >
         <Icon width={16} height={16} icon={GitMerge} />
       </Indicator>
+      <Indicator
```

Comment:
[F] Considered making this the first indicator. Decided against it. Happy to update if you have a different opinion.